x-pack/plugin/ml/build.gradle

```diff
@@ -89,23 +90,28 @@ task extractNativeLicenses(type: Copy) {
 }
 project.afterEvaluate {
```

Comment:
Since lazy task configuration technically defers this until task graph calculation time, could we safely remove the `afterEvaluate` here?

Reply:
the taskProvider also doesn't exist yet. its creation must is deferred.